webapp/web/configure/buttons.css

```diff
@@ -1,64 +1,5 @@
 @charset "utf-8";
```

Comment:
a few of the classes in this file marked for deletion are still used, e.g. `button--confirm`, `button--contained`, `button--add` etc. I suggest we keep this file as is for now, then we transition to using the button class from the common library and delete it then.